config/initializers/assets.rb

```diff
@@ -1,2 +1,2 @@
 Rails.application.config.assets.version = "1.0"
-Rails.application.config.assets.precompile += %w(*.woff2 *.css *.js)
+Rails.application.config.assets.precompile += %w(*.woff2)
```

Comment:
After changed it
I got this error. It's really fine ?  or I am missing something ? 
Thanks 
<img width="1264" alt="2016-10-18 11 43 09" src="https://cloud.githubusercontent.com/assets/5999646/19462549/8fa26e00-9528-11e6-80e6-aa89ffa2ac71.png">